app/components/support_interface/audit_trail_component.rb

```diff
@@ -19,18 +21,29 @@ def audits
         audits = audits.where(auditable_type: params[:auditable_type])
       end
 
-      audits.includes(:user).order('id desc').page(params[:page] || 1).per(60)
+      audits.includes(:user).order('created_at desc').page(params[:page] || 1).per(60)
     end
 
     attr_reader :audited_thing
 
   private
 
-    def audits_for_provider
-      standard_audits.or(Audited::Audit.where(
-                           auditable_type: 'ProviderRelationshipPermissions',
-                           auditable_id: ProviderRelationshipPermissions.where(ratifying_provider_id: audited_thing.id),
-                         ))
+    def provider_audits
+      audited_thing.own_and_associated_audits.unscope(:order).or(
+        Audited::Audit.where(
+          auditable_type: 'ProviderRelationshipPermissions',
+          auditable_id: ProviderRelationshipPermissions.where(ratifying_provider_id: audited_thing.id),
+        ),
+      )
+    end
+
+    def application_audits
+      standard_audits.or(
+        Audited::Audit.where(
+          associated_type: 'ApplicationChoice',
+          associated_id: audited_thing.application_choices.map(&:id),
```

Comment:
```suggestion
          associated_id: audited_thing.application_choices.pluck(:id),
```
This should be a bit more efficient.